test/models/UserModel/test.js

```diff
@@ -0,0 +1,328 @@
+      expect(error).to.be.exist;
+    });
+  });
+  describe('On getAll', async () => {
+    let usersCount;
+    beforeEach(async () => {
+      await dropDatabase();
+    });
+    it('Should returns correct number of records', async () => {
+      usersCount = _.random(1, 10);
+      for (let iter = 0; iter < usersCount; iter++) {
+        await UsersFactory.Create();
+      }
+      const { UserData } = await UserModel.getAll({ skip: 0, limit: usersCount });
+      expect(UserData.length).to.be.eq(usersCount);
+    });
+    it('Should check that the error exists', async () => {
+      const { error } = await UserModel.getAll({ limit: faker.lorem.word() });
+      expect(error).to.be.exist;
+    });
+  });
+  describe('On getObjectsFollow', async () => {
+    let name, countUsers, limit, link;
+    const userName = faker.name.firstName();
+    beforeEach(async () => {
+      await dropDatabase();
+      countUsers = _.random(5, 10);
+      limit = _.random(1, 5);
+      name = faker.name.firstName();
+      link = faker.random.string();
+      await UsersFactory.Create({ name, objects_follow: [link] });
+      await ObjectFactory.Create({ authorPermlink: link });
+      await UsersFactory.Create({ name: userName, objects_follow: [link] });
+    });
+    it('Should return an array', async () => {
+      const { wobjects } = await UserModel.getObjectsFollow({
+        name,
+      });
+      expect(Array.isArray(wobjects)).to.be.true;
+    });
+    it('Should return empty array if username was not transmitted', async () => {
+      const { wobjects } = await UserModel.getObjectsFollow({ limit, skip: 0 });
+      expect(wobjects).to.be.empty;
+    });
+    it('Should check that array isn\'t empty', async () => {
+      const { wobjects } = await UserModel.getObjectsFollow({
+        name: userName,
+      });
+      expect(wobjects.length > 0).to.be.true;
+    });
+    it('Should return object with correct link', async () => {
+      const { wobjects } = await UserModel.getObjectsFollow({
+        name: userName,
+      });
+      expect(wobjects[0].author_permlink).to.be.eq(link);
+    });
+    it('Should return an error if arguments is null', async () => {
+      const { error } = await UserModel.getObjectsFollow(null);
+      expect(error).to.be.exist;
+    });
+  });
+  describe('On aggregate', async () => {
+    let usersCount;
+    beforeEach(async () => {
+      await dropDatabase();
+      usersCount = _.random(1, 10);
+      for (let iter = 0; iter < usersCount; iter++) {
+        await UsersFactory.Create({ name: `asdf_${faker.lorem.word()}` });
+      }
+    });
+    it('Should return array with correct users.length', async () => {
+      await UsersFactory.Create();
+      const { result } = await UserModel.aggregate([
+        {
+          $match: {
+            name: new RegExp('^asdf_'),
+          },
+        }]);
+      expect(Array.isArray(result)).to.be.true;
+      expect(result.length).to.be.eq(usersCount);
```

Comment:
don't use two expect in one it, change Array.isArray, second expect change(result).to.have.length()